Newest TestFile Github Ver/prototype.spawn.js

```diff
@@ -117,14 +127,14 @@ module.exports = function() {
         }
  
    
-       usrSpawn = function(){
+       usrSpawn = function(spawn){
         name = spawn.createClaimer();
        }
 
-       StructureSpawn.prototype.createClaimer = function () {
+       StructureSpawn.prototype.createClaimer = function (creepName) {
             
-         return this.spawnCreep([WORK,CLAIM,CARRY], 'Claimer', { memory: {
-             role: 'claimer', tagret: 'E4N49'
+         return this.spawnCreep([WORK,CLAIM,CARRY, MOVE], creepName, { memory: {
+             role: 'claimer', target: 'E3N48'
```

Comment:
"target' here is still predefined. In spawn memory we should make an array with "claim room" being its name and "claim room toggle" aswell as "claim room name"

visually represented
Claim Room = [
claimRoomToggle,
claimRoomName
]

Reply:
Should we create a new Target Var that will be next room we have our eyes on or?